### ISSUE TITLE
Use TypeScript instead of C# for generator

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [pwa-builder/pwabuilder-ios#32]
What about using TypeScript (Node.js) instead of C# for generator, like https://github.com/GoogleChromeLabs/bubblewrap do?

I truly believe this should significantly lower the contribution threshold for front-end developers. Also it would be really nice to have a CLI in npm for packaging PWA into iOS apps.

BTW, do you have in the team or do you plan to hire an iOS developer to fix iOS-specific bugs in Swift template?

**Comment by FluorescentHallucinogen** [comment]
@JudahGabriel @davrous @jgw96 PTAL. :wink:

**Comment by JudahGabriel** [comment]
Not likely. C# is a great language for servers. 

I built PWABuilder's Android platform in Node and TS, but I still find C# a better tool for server-side jobs. Consider, as an example, that Node doesn't have built-in support for HTTP/2, let alone HTTP/3, and even the mainstream 3rd party libs like `node-fetch` are not fully supporting it at this time. Such shortcomings have caused all kinds of headaches for us and our users.

We certainly considered Node + TS as an alternative. But we decided then, and still believe now, C# is the better tool for the job.

I think most of the open source contributions will come in the Swift template, not in the C#. Indeed, that's the bulk of the PRs we've seen thus far.
